### PR TITLE
Update aws_access_credentials documentation link to credential_type

### LINKED ISSUE #1636: `credential_type` doesn't exist
https://registry.terraform.io/providers/hashicorp/vault/latest/docs/data-sources/aws_access_credentials#ttl refers to `credential_type`, which is mentioned nowhere else on the page and I also can't find where else it might refer to.

This is a documentation issue. The template isn't useful.

---

Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.

### Affected Resource(s)
Please list the resources as a list, for example:
- opc_instance
- opc_storage_volume

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
What should have happened?

### Actual Behavior
What actually happened?

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

### PR DESCRIPTION
**Pull request by HorizonNet**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

Closes #1636

Release note for [CHANGELOG](https://github.com/hashicorp/terraform-provider-vault/blob/master/CHANGELOG.md):

```release-note
Update aws_access_credentials documentation link to credential_type
```

### CODE REVIEW

**Comment by fairclothjm** [comment]
Hi @HorizonNet thanks for the submission, but this is not correct. See the documentation for credential_type here: [credential_type](https://registry.terraform.io/providers/hashicorp/vault/latest/docs/resources/aws_secret_backend_role#credential_type)

**Comment by HorizonNet** [comment]
@fairclothjm Not sure as the issue originally was that the mentioned `credential_type` is not referenced somewhere in the data source `aws_access_credentials`. It is mentioned in the context of the `ttl`, which is only relevant if the `credential_type` of the connected `vault_aws_secret_backend_role` is either `assumed_role` or `federation_token`, but not if the `credential_type` of the `vault_aws_secret_backend_role` is `iam_user`. As the `credential_type` is not mentioned somewhere else on the data source I added a reference to the resource to give a pointer where to look for this one. Or am I missing something else?

**Comment by fairclothjm** [comment]
@HorizonNet I'm sorry, I completely misread this.

File: website/docs/d/aws_access_credentials.html.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ not need to be specified.
 
 * `ttl` - (Optional) Specifies the TTL for the use of the STS token. This
 is specified as a string with a duration suffix. Valid only when
-`credential_type` is `assumed_role` or `federation_token`
+`credential_type` of the connected `vault_aws_secret_backend_role` is `assumed_role` or `federation_token`
```
Comment (fairclothjm) [inline website/docs/d/aws_access_credentials.html.md:91]:
```suggestion
`credential_type` of the connected `vault_aws_secret_backend_role` resource is `assumed_role` or `federation_token`
```

**Review by fairclothjm — COMMENTED** [review]
LGTM, just a small suggestion and I am happy to merge this!

**Comment by HorizonNet** [comment]
@fairclothjm Thanks for taking another look at it. I just updated the PR with your suggestion.

**Review by fairclothjm — APPROVED** [review]
Thanks!

**Pull request merged**